examples/postgres/advanced-blog-cli/src/post.rs

```diff
@@ -0,0 +1,74 @@
+use chrono::NaiveDateTime;
+use auth::User;
+use comment::Comment;
+
+use schema::posts;
+
+#[derive(Queryable, Associations, Identifiable)]
+#[belongs_to(User)]
+pub struct Post {
+    pub id: i32,
+    pub user_id: i32,
+    pub title: String,
+    pub body: String,
+    pub created_at: NaiveDateTime,
+    pub updated_at: NaiveDateTime,
+    pub status: Status,
+}
+
+pub enum Status {
```

Comment:
I'd call more attention to this enum here and its Queryable impl as it's not just boilerplate but actually one of the 'advanced' features :) So maybe add a doc comment that describes how this is used to map `NULL` to `Draft`, and how it's a good idea to do this instead of just using  `Option<NaiveDateTime>`.

Reply:
Advanced querying guide will